comp/core/workloadmeta/def/filter.go

```diff
@@ -19,6 +19,12 @@ type GenericEntityFilterFunc EntityFilterFunc[Entity]
 // any entity.
 func EntityFilterFuncAcceptAll(_ Entity) bool { return true }
 
+// GetNodesMetadata is a filter function that returns true if the metadata
+// belongs to a node.
+var GetNodesMetadata = func(metadata *KubernetesMetadata) bool {
```

Comment:
IMHO the function name is a bit misleading here. 

We already have for example a method called `GetKubernetesMetadata` defined on the workloadmeta interface, which returns the actual metadata object. 

However, in the current context, I think `IsNodesMetadata` is a better name.

Reply:
Changed :+1: